doc/method/sources.md

```diff
@@ -0,0 +1,104 @@
+only released on [date], as this report was being prepared.
+
+Beyond NCBI and GBIF, which together meet the objectives, additional
+development of the taxonomy has been driven unsystematically by
+curator requests.
+
+*SILVA:* Our microbe curators were unhappy about the conventional
+classification of prokaryotes and unicellular Eukaryotes and provided
+a script to import the SILVA taxonomy, which is based on molecular
+evidence.  We incorporated SILVA version 115 into OTT 2.10.  We did
+not include SILVA's plant, animal, or fungi branches in OTT.
+[references]
+
+We gave SILVA higher priority than NCBI or GBIF at curator request.
+
+*Lamiales:* Plant curators requested a revision of order Lamiales based
+on a recent publication about this group.
+[reference, see release notes]
+
+We gave the Lamiales higher priority than NCBI or GBIF at curator request.
+
+*Index Fungorum:* The Open Tree Fungi curators expressed concern over
+the the quality of the combined NCBI/GBIF coverage of Fungi and
+requested we use Index Fungorum.  Around [date] we obtained database
+table dumps for a recent version of IF (the one in GBIF was [XXX]
+years older).  [reference]
+
+We gave the Fungi branch of Index Fungorum higher priority than NCBI
+or GBIF, at curator request.  There was no real call to keep the rest
+of IF, which is not substantial, but we kept it at low priority
+(between NCBI and GBIF) in case it could provide any useful
+information.
+
+*Fungal orders from Hibbett 2007:* The Fungi curators provided a
+higher taxonomy of Fungi down to order.  [reference] We gave this
+taxonomy higher priority than other sources at curator request.  To
+ensure that all of our sources were public, curators deposited this
+taxonomy with Figshare.
+
+*WoRMS:* The Open Tree decapod curators observed that our combined
+taxonomy had only about 80% coverage of decapod species [confirm that
+this is true], and suggested we obtain the decapod taxonomic tree from
+WoRMS.  Given questions over terms of use of the dump files for WoRMS,
+which were only available on request, we obtained the taxonomy via the
+web API, around [date].  [reference]
+
+We gave the decapod branch of WoRMS higher priority than other sources
+at curator request.  Mostly due to unfamiliarity with WoRMS, we felt
+that there was some risk in having the rest of WoRMS also override
+NCBI, so we kept it with lower priority (between NCBI and GBIF).
+
+*IRMNG:* The GBIF backbone did not come with information about whether
+taxa were extinct vs. extant.  (See elsewhere for discussion of the
+importance of this information.)  For extinctness information we
+decided to import IRMNG.  Initial we included all IRMNG records but
+learned through reviewer comments that it was necessary to exclude
+names known to be invalid, which fortunately is information present in
+the IRMNG dump.  [reference]
+
+*Additions:* For various reasons some taxa occurring as OTUs in
```

Comment:
Trying to decide 1) if the additions should be described along with the other patch types; and 2) if additions + patches belong in the sources section, or with the method.